x-pack/test_serverless/functional/test_suites/security/ml/anomaly_detection_jobs_list.ts

```diff
@@ -19,7 +19,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     // Error: Failed to delete all indices with pattern [.ml-*]
     this.tags(['failsOnMKI']);
     before(async () => {
-      await PageObjects.svlCommonPage.loginWithRole(ServerlessRoleName.PLATFORM_ENGINEER);
+      await PageObjects.svlCommonPage.loginWithRole('platform_engineer');
```

Comment:
I think we are safe with hardcoded value to avoid extra imports

Reply:
was it causing a circular dependency issue?

Reply:
yes, importing from `kbn/test-suites-serverless` is the only obvious reason to face this:

```
Project references may not form a circular graph. Cycle detected: /opt/buildkite-agent/builds/bk-agent-prod-gcp-1750917190110240407/elastic/kibana-pull-request/kibana/tsconfig.refs.json
--
kibana/x-pack/platform/plugins/shared/osquery/cypress/tsconfig.type_check.json
kibana/x-pack/test_serverless/tsconfig.type_check.json
kibana/x-pack/test/tsconfig.type_check.json
kibana/x-pack/solutions/observability/test/tsconfig.type_check.json
kibana/x-pack/platform/test/tsconfig.type_check.json
```